### ISSUE TITLE
Fix FP S6747 (`no-unknown-property`): Don't raise for allowed attributes

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4633]
#### path
There are multiple FPs for the `path` element, such as:
- fill-rule
- clip-path
The full permitted attributes are available [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/path).

#### g
The same for the `g` element:
- clip-path
- stop-color
The full permitted attributes are available [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/g).

#### stop

The full permitted attributes are available [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/stop)

```tsx
const LogoIcon = React.forwardRef<SVGSVGElement, LogoIconProps>(
  ({ className, variant = "colour", size = "sm", ...props }, ref) => {
    return (
      <svg
        {...props}
      >
        <g
          clip-path="url(#sourceful-s)"  {/*FP*/}
        >
          <path
            fill-rule="evenodd" {/*FP*/}
            clip-rule="evenodd" {/*FP*/}
            d="M0 12.0001C0 "
            fill={variant === "colour" ? `url(#logoicon-gradient)` : `currentColor`}
          />
        </g>
        <defs>
          <linearGradient
            x1="42"
            y1="4242"
            x2="424242"
            y2="42424242"
            gradientUnits="userSpaceOnUse"
          >
            <stop
              stop-color="#00D0AD" {/*FP*/}
            />
          </linearGradient>
        </defs>
      </svg>
    );
  }
);
```

### css
The `css` attribute is used in [FlowJS](https://github.com/flowjs/flow.js) and [Emotion](https://emotion.sh/docs/introduction).

```js
const Foo = () => {
    const things = [
      {
        label: 'foo',
        value: 'bar',
      },
    ];

    return (
      <>
        {things.map((thing) => {
          return (
            thing.value && (
              <div css={style.flexCell}> {/*FP*/}
                <label css={styles.textLabel}>{thing.label}</label>
                <span css={styles.textValue}>{thing.value}</span>
              </div>
            )
          );
        })}
      </>
    );
};
```

### fetchPriority
It's supported [for all, but Firefox](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority).

```tsx
export function Foo(): JsxElement {
  return (
    <div className="bar" ref={groupRef}>
      <img
         className="foo"
         fetchpriority="low" {/*FP*/}
         decoding="async"
       />
    </div>
  )
}
```

### jsx and global

```tsx
const Foo = (): React.ReactElement => {
  return <>
    <style 
      jsx global {/*FP*/}
    >{` 
      .bar {
        display: block;
        width: 100%;
        height: auto;
      }
    `}</style>
  </>
}
```

### class and variant
Allowed for Stencil

```tsx
import {Component, Event, EventEmitter, h, Host, Prop} from '@stencil/core'

export class Foo {
  render() {
    return (
      <div 
        class={`item-wrapper ${this.errorClass}`} {/*FP*/}
      ></div>
    )
  }
}
```

### Threejs attributes
We raise issues for:
- vertexShader
- uniforms
- defines

which are part of [Three.js ShaderMaterial](https://threejs.org/docs/#api/en/materials/ShaderMaterial).

```tsx
import React from 'react'
import { ShaderMaterialProps } from '@react-three/fiber'

export const Foo = () => (
  <shaderMaterial
    defines={{ PI: Math.PI }} {/*FP*/}
    uniforms={{ // FP
      uAlpha: { value: 1.0 },
    }} 
    vertexShader={webGlTwistVertexShader} {/*FP*/}
  />
)
```

**Comment by Wohops** [comment]
Migrated to Jira. New ticket number: JS-23
